clippy.toml

```diff
@@ -0,0 +1,2 @@
+too-many-arguments-threshold=20 # for what it is worth, clippy is absolutely right and pythonic-ness is absolutely wrong
```

Comment:
Can you clarify what you mean here? Is this an issue with named kwargs or positional args?

Reply:
the *number* of arguments. by default, clippy will warn you that your callable signature is too long if you have more than 7 arguments. 

I've certainly seen **kwargs used for this, but it's not universal. and even if it was, it isn't worth me changing the api